apps/nextjs-website/src/components/PressRelease.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import MarkdownRenderer from './MarkdownRenderer';
+import { PressReleaseSection } from '@/lib/fetch/types/PageSection';
+import { PressRelease as PressReleaseRC } from '@react-components/components';
+import { PressReleaseProps } from '@react-components/types';
+
+const makePressReleaseProps = ({
+  eyelet,
+  title,
+  subtitle,
+  body,
+  ctaText,
+  ctaHref,
+  sectionID,
+}: PressReleaseSection): PressReleaseProps => ({
+  title,
+  eyelet: eyelet || '',
```

Comment:
```suggestion
  ...( eyelet && { eyelet }),
```
Avoid passing empty strings. If a value is null, it shouldn't be rendered at all.